### ISSUE TITLE
Show (Windows) os.version as well

### ISSUE THREAD

**Issue by StefanScherer** [estesp/mquery#5]
It would be nice to also see the OSVersion / os.version field in the output of mquery.

Today I have created a [manifest list spec](https://github.com/StefanScherer/dockerfiles-windows/blob/master/curl/manifest.yml) for the two Windows image version available, one image for Windows Server 2016 LTS channel, and another image for the Windows Server 1709 Semi-annual channel.

You only can run containers with an OS version that matches with the Docker host. (OK, there is also HyperV isolation, but not all cloud providers have nested hypervisors to activate HyperV).

The output of mquery just tells me that it is for amd64/windows.

```
$ docker run mplatform/mquery stefanscherer/curl-windows
Image: stefanscherer/curl-windows
 * Manifest List: Yes
 * Supported platforms:
   - amd64/windows
   - amd64/windows
```

Maybe with `:` or another `/`?

```
$ docker run mplatform/mquery stefanscherer/curl-windows
Image: stefanscherer/curl-windows
 * Manifest List: Yes
 * Supported platforms:
   - amd64/windows:10.0.14393.1770
   - amd64/windows:10.0.16299.19
```

**Comment by estesp** [comment]
Fixed in 0ca13949b486bc25fc9476596152c04311a509f2

However, I had another request to use the canonical form of os/arch/variant (so that ARM/ARM64 matches how containerd shows platform data), and that means windows shows as "windows/amd64". However, I added a colon and then "OSVersion" if it exists to differentiate that it is not a variant of the architecture but another detail of the manifest platform object. Let me know if that seems reasonable.
